rbac/server/api/roles.py

```diff
@@ -14,6 +14,7 @@
 # ------------------------------------------------------------------------------
```

Comment:
How does the sync_direction flag function for when an AD user gets added to an AD group through the NEXT UI/ API endpoints? 

We probably need a sync_direction flag on the update_proposal() endpoint too.